nixos/modules/services/admin/pgadmin.nix

```diff
@@ -1,45 +1,101 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 
 with lib;
```

Comment:
since you already went through the effort of bringing all the `lib` stuff into scope...
```suggestion
```

Reply:
From my (albeit small) experience, people tend to hate `with`. Should I not have `inherit (lib)`'d things?

Reply:
You left the `with lib;` in, I was asking you to remove it.

It's gone now, marking resolved.